tools/webpack-jumpstart

```diff
@@ -85,3 +104,8 @@ fi
 
 tools/node-modules make_package_lock_json
 unpack_from_cache "${tag}"
+
+for file in $(git diff --name-only "${target_tree}"); do
+    message TOUCH "${file}"
+    touch "${file}"
+done
```

Comment:
I don't have a concrete scenario where this would interfere, but somehow it feels wrong/too imprecise to do that. It would render all built object/binary files etc. as out of date as well (I haven't thought much about the implications of touching package.json and friends in the wrong order). I'd much rather do this with `--rebase` only, then at least the damage is more controlled.

Reply:
This actually has more to do with `--best` than with `--rebase` (which should always end up with the identical tree).  In case you don't do `--best` then this will only touch files that you've modified locally and haven't yet committed.

I was having the same thoughts as you, though and went through a couple of different phases there:
 1. what if we're touching files that have some other build dependency order between them?  that could get weird if we touch them in the wrong order
 2. but, it's probably OK, because the built files are all ignored by git
 3. but, regardless, avoiding rebuilding stuff would be nice

What I landed at in terms of the actual proper approach here is to scan the dist/*/Makefile.deps for the changed files and delete the build results for files that have changed.  That's the purest thing we can do.  I was going to save that for some future date, but since you mention it now, I can give it a try :)

Reply:
Sorry, of course I meant `--best` above, that was a typo (fixed). Scanning Makefile.deps and touching the "correct" files in correct order honestly is far beyond what I'd tolerate as "reasonable effort" -- this is rebuilding make rules and `make`. So I'd rather dial this *down* than *up*, and pay the price of occasional webpack build for keeping this reasonably simple and predictable.

Reply:
Dropping this for this PR.  Will bring it back later, with improvements as discussed.